app/src/main/res/layout-v21/topic_play_story_summary.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
```

Comment:
Are we keeping these spaces.

Reply:
No, but this style keeps on shifting based on whose PR got merged last to develop. As of now, I don't think we should focus on this, and this should be solved once hi-fi stuff is finished.